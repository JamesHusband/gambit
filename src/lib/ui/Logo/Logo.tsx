export const Logo = () => (
  <>
    <img
      src="/gambit.png"
      alt="Gambit Logo"
      width={32}
      height={32}
      className="mr-3"
    />
<span className="font-bold text-2xl text-white">Gambit</span>
  </>
);
