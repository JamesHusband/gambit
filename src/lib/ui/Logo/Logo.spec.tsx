import { render, screen } from "@testing-library/react";
import { Logo } from "./Logo";

describe("Logo", () => {
  it("should render", () => {
    const { container } = render(<Logo />);
    expect(container).toMatchSnapshot();
  });
  it("should render logo with text", () => {
});
