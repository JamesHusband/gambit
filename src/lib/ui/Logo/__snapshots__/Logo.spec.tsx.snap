// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Logo should render 1`] = `
<div>
  <img
    alt="Gambit Logo"
    class="mr-3"
    height="32"
    src="/gambit.png"
    width="32"
  />
  <span
    class="font-bold text-2xl text-white"
  >
    Gambit
  </span>
</div>
`;
